tutorxqueue/templates/xqueue/build/xqueue/Dockerfile

```diff
@@ -21,9 +21,9 @@ WORKDIR /openedx/xqueue
 RUN python -m venv /openedx/venv
 ENV PATH /openedx/venv/bin:${PATH}
 
-RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/openedx/.cache/pip,sharing=shared {% endif %}pip install --upgrade pip setuptools
+RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/openedx/.cache/pip,sharing=shared {% endif %}pip install setuptools==68.2.2 pip==23.2.1. wheel==0.41.2
```

Comment:
Good catch. Could you please add links as comments to the pypi projects, just like in the other plugins?